scrapers/czechcasting.yml

```diff
@@ -0,0 +1,25 @@
+name: czechav
```

Comment:
Just noticed that `czechav` name is used from the `czechav.yml` scraper.
Can you rename to `czechcasting` or something else?